packages/makeup-hoist-element/index.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+var modalEl;
+var modalPlaceholder;
+var containerDiv;
+var prevAriaHiddenSetting;
+var bodyIndexes = [];
+
+function unhoist() {
+  if (modalEl) {
+    if (containerDiv) {
+      var childList = Array.from(containerDiv.childNodes);
+      childList.forEach(function (child) {
+        if (child.src === undefined) {
+          var index = bodyIndexes.shift();
+
+          if (index > document.body.childNodes.length) {
+            document.body.appendChild(child);
+          } else {
+            document.body.insertBefore(child, document.body.childNodes[index + 1]);
+          }
+        }
+      });
+      containerDiv.remove();
+      containerDiv = null;
+      bodyIndexes = [];
+    }
+
+    if (modalPlaceholder) {
+      modalEl.removeAttribute('aria-hidden'); // eslint-disable-next-line eqeqeq
+
+      if (prevAriaHiddenSetting != null) {
```

Comment:
Two questions:
1. Why disable `eqeqeq`?
2. Is the hoisting component the one that's going to handle `aria-hidden`? I thought there was another makeup module to handle that?

Reply:
This module should not do anything with aria-hidden. Only makeup-screenreader-trap should.

Reply:
I disabled `eqeqeq` because I wanted to check for null or undefined. But this line will get deleted anyways.